### PR TITLE
fix: created btc dust amount constant and renaming

### PR DESCRIPTION
**Pull request by ws4charlie**
# Description

The temporary hotfix for dust btc amount is in zetaclient. The fix needs to be done in zetacore to prevent dust CCTX from registration (https://github.com/zeta-chain/node/issues/2326). The PR only ported the constant and remaining.

- Renamed `BtcDustOffset` -> `BtcNonceMarkOffset` and created `BTCWithdrawalDustAmount` constant for zetacore usage.

Closes: <PD-XXXX>

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Include instructions and any relevant details so others can reproduce. 

- [ ] Tested CCTX in localnet
- [ ] Tested in development environment
- [ ] Go unit tests
- [ ] Go integration tests
- [ ] Tested via GitHub Actions 

# Checklist:

- [ ] I have added unit tests that prove my fix feature works

### CODE REVIEW

**Review by lumtis — COMMENTED** [review]
It doesn't seem the change modify any logic in ZetaClient did we forget to do 
```
2000+BTCWithdrawalDustAmount
```
?

**Comment by ws4charlie** [comment]
> It doesn't seem the change modify any logic in ZetaClient did we forget to do
> 
> ```
> 2000+BTCWithdrawalDustAmount
> ```
> 
> ?

The 2nd commit: https://github.com/zeta-chain/node/commit/b40b511cf1856cc24e4fcfdc8b68b4206a988c08
The 2nd commit according to Panruo's comment: https://github.com/zeta-chain/node/commit/8a34fbe7d602865451fb41eb0eaa9564bba8a849

File: pkg/constant/constant.go [hunk 1]
```diff
@@ -10,4 +10,7 @@ const (
 
 	// CmdMigrateTssFunds is used for CCTX of type cmd to give the instruction to the TSS to transfer its funds on a new address
 	CmdMigrateTssFunds = "cmd_migrate_tss_funds"
+
+	// BTCWithdrawalDustAmount is the minimum satoshis that can be withdrawn from zEVM to avoid outbound dust output
+	BTCWithdrawalDustAmount = 1000
```
[the remark this answers is not in the document]
Comment (ws4charlie) [reply pkg/constant/constant.go:15]: thanks, changelog added: https://github.com/zeta-chain/node/pull/2327/commits/1d3a2e9ef7d0ebb15ab8280deac4168eb52e8ac6

[the remark this answers is not in the document]
Comment (ws4charlie) [reply pkg/constant/constant.go:15]: This is a partial cherry-pick as the hotfix is a temporary fix. The new fixing task is tracked by https://github.com/zeta-chain/node/issues/2326

**Review by lumtis — APPROVED** [review]

**Comment by lumtis** [comment]
Does this fix https://github.com/zeta-chain/node/issues/2326?

**Pull request merged**